tests/contrib/grpc_aio/test_grpc_aio.py

```diff
@@ -1005,3 +1035,130 @@ async def test_client_streaming(server_info, tracer):
     out, err, status, _ = ddtrace_run_python_code_in_subprocess(code, env=env)
     assert status == 0, (err.decode(), out.decode())
     assert err == b"", err.decode()
+
+
+class StreamInterceptor(grpc.aio.UnaryStreamClientInterceptor):
+    async def intercept_unary_stream(self, continuation, call_details, request):
+        response_iterator = await continuation(call_details, request)
+        return response_iterator
+
+
+async def run_streaming_example(server_info, use_generator=False):
+    i = 0
+    async with grpc.aio.insecure_channel(server_info.target, interceptors=[StreamInterceptor()]) as channel:
+        stub = MultiGreeterStub(channel)
+
+        # Read from an async generator
+        if use_generator:
+            async for response in stub.sayHello(HelloRequestStream(name="you")):
+                assert response.message == "Hello number {}, you!".format(i)
+                i += 1
+
+        # Direct read from the stub
+        else:
+            hello_stream = stub.sayHello(HelloRequestStream(name="will"))
+            while True:
+                response = await hello_stream.read()
+                if response == grpc.aio.EOF:
+                    break
+                assert response.message == "Hello number {}, will!".format(i)
+                i += 1
+
+
+@pytest.mark.asyncio
+@pytest.mark.skip("Bug/error from grpc when adding an async streaming client interceptor throws StopAsyncIteration")
```

Comment:
Can we link to the upstream issue for this?

Reply:
ok done